bazel/BUILD

```diff
@@ -19,3 +19,8 @@ kt_jvm_library(
 exports_files([
     "pom_template.xml",
 ])
+
+alias(
+    name = "zipper",
+    actual = "@bazel_tools//tools/zip:zipper",
```

Comment:
I'd probably just inline this

Reply:
buildifier (or an envoy format tool) complains that I'm using `@bazel` directly if I do